new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/InstallWITCommon.java

```diff
@@ -42,17 +36,17 @@ protected void checkDirectory(String dir) {
   }
 
   /**
-   * Check if the required directoies exist.
-   * Currently the directies will be created if missing. We may remove this function
-   * once we have all required working directies pre-created.
+   * Check if the required directories exist.
+   * Currently the directories will be created if missing. We may remove this function
```

Comment:
the code does not appear to create any directories?

Reply:
fixed the javadoc